docs/guide/acquiring_data/fido.rst

```diff
@@ -134,8 +135,8 @@ operator would::
                                                    ...                  keV
```

Comment:
this example has a `+SKIP` on it, which I hope can be removed now?

Reply:
yep